Frontend/ui-customer/src/Screens/TicketBookingFlow/MetroTicketBooking/View.purs

```diff
@@ -60,14 +66,18 @@ screen initialState =
   , globalEvents : [getQuotes]
   , eval : \action state -> do
         let _ = spy "MetroTicketBookingScreenState action " action
-        let _ = spy "MetroTicketBookingScreenState state " state
+        let _ = spy "MetroTicketBookingScreenState state " state 
+        case action of
+          ShowMetroBookingTimeError _ -> pure unit
```

Comment:
@spravinkumar9952 Why was this required `ShowMetroBookingTimeError` ?

Reply:
yes. otherwise pooling will stop